EssentialCSharp.Web/Helpers/SitemapXmlHelpers.cs

```diff
@@ -0,0 +1,137 @@
+using DotnetSitemapGenerator;
+using DotnetSitemapGenerator.Serialization;
+using Microsoft.AspNetCore.Mvc.Infrastructure;
+
+namespace EssentialCSharp.Web.Helpers;
+
+public static class SitemapXmlHelpers
+{
+    private const string RootUrl = "https://essentialcsharp.com/";
```

Comment:
Do we want this hard coded? Often we can access this via `IHttpContextAccessor` or `IServer` or similar.

Reply:
Since we are in docker container, just making configurable from appsettings